docs/docs/integrations/vectorstores/pathway.ipynb

```diff
@@ -230,53 +144,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Further Configuration"
+    "## Your own pipeline"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Parsing Documents\n",
-    "\n",
-    "The vectorization pipeline supports pluggable parsers. If not provided, defaults to `UTF-8` parser. You can find available parsers [here](https://github.com/pathwaycom/pathway/blob/main/python/pathway/xpacks/llm/parser.py).\n",
+    "### Running in production\n",
+    "To have your own Pathway data indexing pipeline check the Pathway's offer for [hosted pipelines](https://pathway.com/solutions/ai-pipelines). You can also run your own Pathway pipeline - for information on how to build the pipeline refer to [Pathway guide](https://pathway.com/developers/user-guide/llm-xpack/vectorstore_pipeline/). You can use [Langchain components](https://pathway.com/developers/user-guide/llm-xpack/vectorstore_pipeline/#langchain) Pathway for processing documents. \n",
     "\n",
-    "An example parser that can read PDFs, Word documents and other formats is provided with `parsers.ParseUnstructured`: "
+    "A production deployment will typically run the server in a separate process. We recommend to run the Pathway data indexing pipeline in a container-based deployment environment, such as Docker or Kubernetes. For more info see [Pathway's deployment guide](https://pathway.com/developers/user-guide/deployment/)."
```

Comment:
I'd also remove it, and make sure it is in the docs for the pathway vector pipeline - lets keep these langchain docs speaking about possibilities, but not necessairly explaining them (we show what is doable, but not how)